clarity/contracts/pool/alex-reserve-pool.clar

```diff
@@ -21,19 +23,45 @@
 (define-constant ERR-EXPIRY (err u2017))
 (define-constant ERR-GET-BALANCE-FAIL (err u6001))
 (define-constant ERR-NOT-AUTHORIZED (err u1000))
+(define-constant ERR-TRANSFER-FAILED (err u3000))
```

Comment:
Is there a central registry for all ALEX error codes? If not then it is advisable to start working on one.

Reply:
https://docs.alexgo.io/developers/smart-contracts/error-codes
@hyossid @JBotwina are updating them.